Source/WTF/wtf/cocoa/VectorCocoa.h

```diff
@@ -118,7 +119,17 @@ inline Vector<uint8_t> makeVector(NSData *data)
     return span(data);
 }
 
+template<typename T>
+inline RetainPtr<dispatch_data_t> dispatchDataFromVector(Vector<T>&& vector)
```

Comment:
Not sure that I’m a fan of having the words _from vector_ in the function name. I would call this `makeDispatchData`. Doesn’t seem likely that it would compile if it was misused by a caller that’s not moving a `Vector`. I like to use the word _make_ rather than _create_ for functions that safely create something without potentially leaking because they use a smart pointer or other object that manages storage.

Reply:
Not sure about my _make_ vs. _create_ comment, on reflection. Either might be good.